test/data/javascript/tests.js

```diff
@@ -169,6 +169,22 @@ exports.test_data = {
           ]
         }
       ]
+    }, {
+      name: "ES5 Class Private Fields",
+      description: "Permit ES5 private class fields which are declared with a leading \"#\".",
+      tests: [
+        { unchanged: '#foo' },
+        {
+          unchanged: [
```

Comment:
Please add some tests where the input and output differ.  Then this is will be good to go.

Reply:
Added.